routing/cross_mwm_connector.hpp

```diff
@@ -294,6 +299,7 @@ class CrossMwmConnector final
   }
 
   NumMwmId const m_mwmId;
+  uint32_t const m_featureNumerationOffset;
```

Comment:
`uint32_t const m_featureNumerationOffset = 0;` and removing from ctor `m_featureNumerationOffset(0)`?

Reply:
done